frontend/src/components/Sidepanel/Panels/Modules/moduleWindowStyling.ts

```diff
@@ -0,0 +1,147 @@
+import { styled } from 'goober'
+import { forwardRef } from 'react'
+
+export const ModuleWindowWrapper = styled('div', forwardRef)<{ width: string }>`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: ${(props) => props.width};
+  background-color: var(--surface);
+  border-right: 1px solid var(--surface);
+  color: var(--white);
+  position: relative;
+  overflow-y: auto;
+  overflow: visible; 
+  padding: 0 1em 1em;
+`
+
+export const ResizeHandle = styled('div')`
+  height: 100%;
+  background-color: rgb(165, 165, 165);
+  width: 2px;
+  position: absolute;
+  top: 0;
+  right: 0;
+  cursor: ew-resize;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+  transition: background-color 0.2s ease-out, width 0.2s ease-out;
+
+  &:hover {
+    width: 3px;
+    background-color: rgb(93, 159, 235);
+  }
+`
+
+export const CloseButton = styled('button')`
+  position: absolute;
+  top: .6em;
+  right: -1em;
+  height: 2em;
+  width: 2em;
+  background: var(--toolbar);
+  border: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: .3em;
+  cursor: pointer;
+`
+
+export const TitleWrapper = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+`
+
+export const Title = styled('h2')`
```

Comment:
some of these seem very generic. Like there is already a `Title` element defined elsewhere https://github.com/automatarium/automatarium/blob/660e3af621efc99095f14995f30684e5cf015a4f/frontend/src/pages/Tutorials/tutorialsStyle.ts#L10

Would some existing components work instead of defining them again?